src/olympia/devhub/tasks.py

```diff
@@ -307,7 +308,6 @@ def forward_linter_results(results, upload_pk):
     linter results to `handle_upload_validation_result()` (the callback of the
     chord).
     """
-    log.info('Called forward_linter_results() for upload_pk = %d', upload_pk)
```

Comment:
That was kinda useful to me, but maybe it could be a `.debug()`?

Reply:
76311339e9f67e0ea992ab2e466713c94945bc65